src/app/components/BookCell/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Text, View, Image, TouchableOpacity } from 'react-native';
+import { Book } from '@interfaces/book';
+import placeholder from '@assets/img_book6.png';
+
+import styles from './styles';
+
+interface Props {
+  book: Book;
+  onBookPress: (book: Book) => void;
+}
+
+function BookCell({ book, onBookPress }: Props) {
+  return (
+    <TouchableOpacity onPress={() => onBookPress(book)} style={styles.container}>
+      <Image
+        source={
+          book.imageUrl
+            ? {
+                uri: book.imageUrl
```

Comment:
You can destructure the book props to avoid repeating, for example, `book.imageUrl`
```
const { imageUrl, title, author } = book;
return (
...
);
```